Classes/Private/Manager/Submanagers/OCTCallsContainer.m

```diff
@@ -0,0 +1,118 @@
+//
+//  OCTCallsContainer.m
+//  objcTox
+//
+//  Created by Chuong Vu on 6/10/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "OCTCallsContainer.h"
+#import "OCTCallsContainer+Private.h"
+#import "OCTBasicContainer.h"
+
+@interface OCTCallsContainer () <OCTBasicContainerDelegate>
+
+@property (strong, nonatomic) OCTBasicContainer *container;
+
+@end
+
+@implementation OCTCallsContainer
+
+#pragma mark - Lifecycle
+
+- (instancetype)init
+{
+    self = [super init];
+
+    if (! self) {
+        return nil;
+    }
+
+    self.container = [[OCTBasicContainer alloc] initWithObjects:nil];
+    self.container.delegate = self;
+
+    return self;
+}
+#pragma Public Methods
+
+/**
+ * @return Number of calls
+ */
+- (NSUInteger)numberOfCalls
+{
+    return self.container.count;
+}
+
+/**
+ * Returns call at specified index.
+ * @param index Index to get call.
+ * @return Call object at index. Nil if out of bounds.
+ */
+- (OCTCall *)callAtIndex:(NSUInteger)index
+{
+    return [self.container objectAtIndex:index];
+}
+
+/**
+ * Returns call for specified friend.
+ * @param friend Friend to look for call
+ * @return Call object of that contains friend. Nil if not found.
+ */
+- (OCTCall *)callWithFriend:(OCTFriend *)friend
```

Comment:
I'm not sure if we need `callWithFriend` method. `callWithChat:` would be more useful. It will be groundwork for the future, when we'll have group calls. API will remain same - you don't care how much there are friends, you just specify a chat.

Currently OCTChat doesn't have `isEqual:` method, however we can provide one. I was thinking about adding `uniqueIdentifier` and `isEqual:` methods to all objects.
So you can add it to OCTChat if you like, and I'll add it to another objects later.